frontend/src/layout/navigation-3000/components/ProjectTree/projectTreeLogic.tsx

```diff
@@ -96,6 +107,17 @@ export const projectTreeLogic = kea<projectTreeLogicType>([
                 loadUnfiledItemsFailure: (state) => state - 1,
             },
         ],
+        loadingTree: [
+            true, // Start as true since we load items on mount
+            {
+                loadSavedItems: () => true,
+                loadSavedItemsSuccess: () => false,
+                loadSavedItemsFailure: () => false,
+                loadUnfiledItems: () => true,
+                loadUnfiledItemsSuccess: () => false,
+                loadUnfiledItemsFailure: () => false,
+            },
+        ],
```

Comment:
This should be replaced with a selector `loading` that depends on `unfiledItemsLoading` and `savedItemsLoading`